packages/api/src/__tests__/server.spec.js

```diff
@@ -57,3 +58,89 @@ describe('The API server', () => {
     })
   })
 })
+
+jest.mock('winston', () => ({
```

Comment:
If this is stubbing the logger do we want to do that for any tests ? 

we can use manual mocks to achieve this - https://jestjs.io/docs/manual-mocks. 

But i am not sure on how the logger is used in the actual routes ?

Reply:
Or this could be a spy and let the real thing work ?

Reply:
When unit testing I think you should only ever be testing the code that you've written yourself. In this case I wanted to test that the `logResponse` and `handleErrors` functions make the correct function calls and return the desired value. My limiting the scope of the tests was intentional.

I don't think we should be writing the tests to test that the `winston` package works, nor any other function dependancies that are covered elsewhere.

I'm also trying to be consistent with the pre-existing pattern that is prominent throughout the codebase's other unit tests.